remotespark/utils/jupyterevents.py

```diff
@@ -0,0 +1,28 @@
+# Read the config file that will return the filehandler
+# file handler will be one of two local file handler or hdinsighteventshandler
```

Comment:
No mention of hdinsight in this repo please. Just say that the file handler will be one of N implementations, default one being X.